### PR TITLE
[s] nerfs lollerderby

### PR DESCRIPTION
**Pull request by PKPenguin321**
i warned you not to talk shit with coders :ok_hand:

### CODE REVIEW

**Comment by Durkel** [comment]
Sick meme

**Comment by Aloraydrel** [comment]
I've been cheated for the last time

**Comment by lollerderby** [comment]
p-pls

**Comment by Core0verload** [comment]
Did you tested it? You want to be really fucking sure it works before pushing it. Grudgecoding is a serious business.

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 
 /mob/living/carbon/human/New()
+	if(ckey = lollerderby)//don't test me
+ 		Destroy()
```
Comment (duncathan) [inline code/modules/mob/living/carbon/human/human.dm]: Hilariously, this won't do anything
You gotta do `qdel(src)`

Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/human.dm]: hey man dont blame me i stole this code from hornygranny. https://github.com/tgstation/-tg-station/pull/7078

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -14,6 +14,10 @@
 
 
 /mob/living/carbon/human/New()
+	if(ckey = lollerderby)//don't test me
```
Comment (duncathan) [inline code/modules/mob/living/carbon/human/human.dm]: Oh yeah and this won't compile

Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/human.dm]: B l a m e H G
l
a
m
e
H
G

**Pull request closed without merging**